packages/react-components/source/react/library/icon/icons.js

```diff
@@ -399,6 +399,16 @@ const icons = {
     ),
   },
 
+  custom: {
```

Comment:
I just chatted with @jtmuz and we decided on a more descriptive name for this icon:

```suggestion
  'pencil-list': {
```